src/runJupyterDaskOnSLURM.py

```diff
@@ -44,23 +44,26 @@
                   ATTENTION! When adding a job script on a plattform other than spider/snellius @SURF the user MUST
                   create a job script for the platform (this can be done by using the existing scripts as templates).
                   Expected naming convention is 'jupyter_dask_xxx.bsh', where xxx is the name of the platform.
-""" 
+"""
 
 
 import argparse
 import configparser
 import math
-import os
 import time
 import webbrowser
 import getpass
 
 from fabric import Connection
+from pathlib import Path
 import installJDOnSLURM     #Functions to install or uninstall JDOnSLURM
 
-config_path = './config/platforms/platforms.ini'
+# get the parent pat of the current file
+parent_path = Path(__file__).resolve().parent
```

Comment:
By adding one level to the dir above, I guess this would need to be updated?

Reply:
I replaced it with `config_path = Path.home() / ".config/platforms/platforms.ini"` and in the user-guide.md, I asked users to create it.